lib/ccf/structural_preprocessing/one_subject_job_submitter.py

```diff
@@ -562,13 +569,112 @@ def mark_running_status(self, stage):
             mark_cmd += os.sep + self.PIPELINE_NAME 
             mark_cmd += os.sep + self.PIPELINE_NAME
             mark_cmd += '.XNAT_MARK_RUNNING_STATUS' 
+            mark_cmd += ' --user=' + self.username
```

Comment:
I think these changes in the mark_running_status method are part of this "change location of running status" change set.